api-report/cell.api.md

```diff
@@ -35,8 +35,8 @@ export interface ISharedCellEvents<T> extends ISharedObjectEvents {
 // @public
 export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> implements ISharedCell<T> {
     constructor(id: string, runtime: IFluidDataStoreRuntime, attributes: IChannelAttributes);
-    // (undocumented)
-    protected applyStashedOp(): void;
+    // @internal (undocumented)
+    protected applyStashedOp(content: unknown): number;
```

Comment:
Nit: might be better to explicitly return `unknown` here rather than letting typescript infer `number`, since if it's exposed here then changing it would technically be a breaking change even though the localOpMetadata type is internal.